### ISSUE TITLE
New integration for sys.unraisablehook

### ISSUE THREAD

**Issue by graingert** [getsentry/sentry-python#374]
Python 3.8 has a new hook: sys.unraisablehook https://github.com/python/cpython/pull/13187

Sentry should set this to be able log details about 'unraisable' exceptions

**Comment by untitaker** [comment]
Yeah this should be doable. I'd imagine a lot of errors in `__del__` are when the interpreter shuts down though, and I don't know if we're capable to send to Sentry that late.

**Comment by graingert** [comment]
!

**Comment by antonpirker** [comment]
Looks like a nice to have enhancement. Putting it in the internal backlog at low priority.

@graingert Any help and PRs for this are very welcome!

**Comment by youtux** [comment]
hi, is there any progress on this? I'm getting errors from gRPC that cause Celery tasks to fail, but unfortunately they are not sent to Sentry and we have no visibily to such failed tasks.

**Comment by antonpirker** [comment]
No news about this right now. But PRs are always welcome if you want to give it a try @youtux !

**Comment by antonpirker** [comment]
This should be a new integration that is similar to `ExcepthookIntegration` and enabled by default.
